app/post_metadata.go

```diff
@@ -402,7 +403,10 @@ func (a *App) getLinkMetadata(requestURL string, timestamp int64, isNewPost bool
 	}
 
 	if body != nil {
-		defer body.Close()
+		defer func() {
+			io.Copy(ioutil.Discard, body)
+			body.Close()
```

Comment:
0/5 Maybe move this into a helper function similar to the one in `services/marketplace/client.go`?

Reply:
This is slightly different because it works on the Body directly rather than from the request. So it will be a one-off function. Let's keep it here for now.